### PR TITLE
Add clarification to quota docs regarding parent ns access

### PR DESCRIPTION
**Pull request by ccapurso**
Creating, updating, and deleting quotas within a parent or sibling namespace is not allowed. Reading and listing, however, is. This PR introduces a few small documentation changes to clarify this difference for both rate limit and lease count quota API endpoints.

### CODE REVIEW

File: website/content/api-docs/system/lease-count-quotas.mdx [hunk 1]
```diff
@@ -131,8 +137,10 @@ $ curl \
 
 ## List lease count quotas
 
-This endpoint returns a list of all the lease count quotas. A 404 response will
-be returned if no lease count quota has been created.
+This endpoint returns a list of all the lease count quotas across all namespaces.
+The global visibility differs from the restricted access enforced when modifying
```
Comment (divyaac) [inline website/content/api-docs/system/lease-count-quotas.mdx:141]: Might be my brain but this one was a hard for me to absorb on first glance. Maybe reword this a bit?

Comment (ccapurso) [reply website/content/api-docs/system/lease-count-quotas.mdx]: I agree, thanks for the feedback. I took a stab at rewording this in https://github.com/hashicorp/vault/pull/25894/commits/2c747d7c58d9ab3db3e0f78a8f76869c489ff3e9.

File: website/content/api-docs/system/rate-limit-quotas.mdx [hunk 2]
```diff
@@ -129,7 +134,9 @@ $ curl \
 
 ## List rate limit quotas
 
-This endpoint returns a list of all the rate limit quotas.
+This endpoint returns a list of all the lease count quotas across all namespaces.
```
Comment (ccapurso) [inline website/content/api-docs/system/rate-limit-quotas.mdx:137]: **NOTE:** This is a typo and should be "rate limit" instead of "lease count." I unfortunately did not catch it until after merging. A new https://github.com/hashicorp/vault/pull/25899 has been opened to correct it on main. Once merged, it will be backported accordingly.
